src/commands/force/org/beta/create.ts

```diff
@@ -23,12 +24,22 @@ import {
   SandboxUserAuthResponse,
   SfdxError,
   StatusEvent,
+  ScratchOrgRequest,
+  ScratchOrgInfo,
 } from '@salesforce/core';
+import { OrgCreateResult } from '../../../../shared/orgHooks';
 import { SandboxReporter } from '../../../../shared/sandboxReporter';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-org', 'create');
 
+export interface ScrathcOrgProcessObject {
```

Comment:
I checked the name and fixed it, don't know ahout the `Object` end I just wanted to keep it similar to @WillieRuemmele code but I can change it, it would be ideal if both sandbox and scratch org share a similar naming convention, what you think @WillieRuemmele  ?